microsoft-edge/extensions/publish/add-ons-badge.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Use the Microsoft Edge Add-ons badge to promote your add-on
+description: Use this badge in your websites, documentation, or tutorials, to promote your extension (or theme) that's at the Microsoft Edge Add-ons site.  # key words before col 158
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.date: 09/15/2025
+---
+# Use the Microsoft Edge Add-ons badge to promote your add-on
+<!-- upstream:
```

Comment:
I don't think we need this comment anymore. We've deviated from this "upstream" doc, and the upstream doc is no longer available anyway.